src/ScheduleService.Data/GroupRepository.cs

```diff
@@ -0,0 +1,35 @@
+﻿using LT.DigitalOffice.ScheduleService.Data.Interfaces;
+using LT.DigitalOffice.ScheduleService.Models.Db;
+using LT.DigitalOffice.ScheduleService.Models.Dto.Requests.Group;
+using System;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.ScheduleService.Data;
+
+public class GroupRepository : IGroupRepository
```

Comment:
you don't have method for patching

Reply:
is public Task<bool> EditAsync(Guid groupId, EditGroupRequest request) not suitable?

Reply:
I mean in repository not another command